ci-operator/step-registry/assisted/vsphere/teardown/machine/assisted-vsphere-teardown-machine-ref.yaml

```diff
@@ -4,7 +4,7 @@ ref:
   commands: assisted-vsphere-teardown-machine-commands.sh
   credentials:
   - namespace: test-credentials
-    name: vsphere-template-ssh-keys
+    name: assisted-installer-ci-ssh-keys
```

Comment:
Will you follow up with oci as well? If not, feel free to create a ticket and assign it to me

Reply:
That was the plan, After this one will merge we will use this SSH pair in all of our platforms (excepts baremetal for now).
I'll create a ticket for it